src/components/UserProfile.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { Button, Card, CardBody, CardFooter, Table } from 'reactstrap';
```

Comment:
If we're using reactstrap as well, you should make sure to install `reactstrap` so that it's in `package.json` otherwise there will be errors when others try to run this.